homeassistant/components/sabnzbd/sensor.py

```diff
@@ -127,9 +129,16 @@ def __init__(
         self, sabnzbd_api_data, client_name, description: SabnzbdSensorEntityDescription
     ):
         """Initialize the sensor."""
+        unique_id = description.key
```

Comment:
Only the key is not unique enough if more than one config entry is setup. We could prefix with the config entry entry_id if there's no better unique_id.

https://developers.home-assistant.io/docs/entity_registry_index#unique-id-requirements

Reply:
https://github.com/home-assistant/core/pull/71455